transcom/mymove
# B 18882 b add search capability for too and tio

[##](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/848240) [B-18882](tbd)

## Summary

Added capability for TOO and TIO to be able to search for Moves

### How to test

TIO:
1. Login as TOO
2. pick a Move Code, DOD ID, and Name from the first queue for later
3. Click the Search tab
4. Run a search by default (customer name) using the name from earlier, only moves with that name should show
5. Click on status (Verify that these SUBMITTED, APPROVALS REQUESTED, APPROVED exist)
6. Click on one or any combination to filter down to the move statuses as shown
5. Click on any of the move(s) that have shown after filtering
7. Go back to the search queue
8. Use the move code from earlier to run another search, verify only moves with that move code show
9. Run the move again using DOD ID as an option, only moves with this ID should show.
10. Try filtering without anything specified in all 3 options (shouldn't work)

TIO:
1. Login as TIO
2. pick a Move Code, DOD ID, and Name from the first queue for later
3. Click the Search tab
4. Run a search by default (customer name) using the name from earlier, only moves with that name should show
5. Click on status (Verify that these Payment requested, Reviewed, Rejected, Paid, Deprecated, Error exist)
6. Click on one or any combination to filter down to the Payment Request statuses as shown
5. Click on any of the move(s) that have shown after filtering
7. Go back to the search queue
8. Use the move code from earlier to run another search, verify only moves with that move code show
9. Run the move again using DOD ID as an option, only moves with this ID should show.
10. Try filtering without anything specified in all 3 options (shouldn't work)
